jupyterlab/staging/package.json

```diff
@@ -114,18 +114,18 @@
     "@jupyterlab/vdom": "~4.0.0-alpha.12",
     "@jupyterlab/vdom-extension": "~4.0.0-alpha.12",
     "@jupyterlab/vega5-extension": "~4.0.0-alpha.12",
-    "@lumino/algorithm": "^1.9.1",
-    "@lumino/application": "^1.27.0",
-    "@lumino/commands": "^1.19.0",
-    "@lumino/coreutils": "^1.11.1",
-    "@lumino/disposable": "^1.10.1",
-    "@lumino/domutils": "^1.8.1",
-    "@lumino/dragdrop": "^1.13.1",
-    "@lumino/messaging": "^1.10.1",
-    "@lumino/properties": "^1.8.1",
-    "@lumino/signaling": "^1.10.1",
-    "@lumino/virtualdom": "^1.14.1",
-    "@lumino/widgets": "^1.33.0",
+    "@lumino/algorithm": "^2.0.0-alpha.1",
+    "@lumino/application": "^2.0.0-alpha.1",
+    "@lumino/commands": "^2.0.0-alpha.1",
+    "@lumino/coreutils": "^2.0.0-alpha.1",
+    "@lumino/disposable": "^2.0.0-alpha.1",
+    "@lumino/domutils": "^2.0.0-alpha.1",
+    "@lumino/dragdrop": "^2.0.0-alpha.1",
+    "@lumino/messaging": "^2.0.0-alpha.1",
+    "@lumino/properties": "^2.0.0-alpha.1",
+    "@lumino/signaling": "^2.0.0-alpha.1",
+    "@lumino/virtualdom": "^2.0.0-alpha.1",
+    "@lumino/widgets": "^2.0.0-alpha.1",
```

Comment:
You should undo those changes. They are the reason the usage and interop test are failing.